atomic.d/openscap

```diff
@@ -1,6 +1,6 @@
 type: scanner
 scanner_name: openscap
-image_name: openscap
+image_name: registry.access.redhat.com/rhel7/openscap
 default_scan: cve
```

Comment:
On the other hand, not doing so could allow someone to spoof and image in there.  You decide, I'll change it.